main/webapp/modules/core/scripts/reconciliation/standard-service-panel.html

```diff
@@ -1,11 +1,15 @@
 <div class="recon-dialog-service-panel recon-dialog-standard-service-panel">
   <div class="grid-layout layout-normal layout-full grid-layout-for-ui"><table>
     <tr>
+      <td colspan="6" style="text-align: right;">
```

Comment:
Lots of services does not have a documentation link and we must handle that as well, as it is currently it would just create a broken link for services missing the property.

Reply:
Hey, here is the screenshot of how i handled this 
![Screen Shot 2023-04-18 at 3 12 41 AM](https://user-images.githubusercontent.com/72227873/232617901-7febfffa-8cb2-44b6-95d9-21907223005d.png)
it works well!

Reply:
I would rather not display any link if there is no documentation URL available.

Reply:
I have hidden the link if there is no documentation url available, please see the screenshot in above conversation.

Reply:
i am sorry, will not upload screenshots from now onwards, also i pointed you towards the wrong screenshot, i meant the screenshot in https://github.com/OpenRefine/OpenRefine/pull/5784#discussion_r1170481268 conversation, but leave, it i am gonna do a pr right away .